### ISSUE TITLE
Listing Request DTCN

### ISSUE THREAD

**Issue by dtcn** [decentrex/decentrex.github.io#43]
Hi DECENTREX Team,
I would like to request to add officially DCoin (DTCN) on DECENTREX.
About the coin:
DCoin is a virtual digital currency project established on the basis of Decentralization method, characterized with fluency, decentralization and transparency. DCoin (DTCN) is an ERC20 token, works under the Ethereum ecosystem and DCoin (DTCN) is faster ERC20 in terms of block generation time plus the privacy; with DCoin we works faster and secure in term of speed and privacy.
Token details:
Verified on Etherscan:
https://etherscan.io/token/0x1a86b24ca84aa7049a974bb6db83f0bd80c60713
Contract: 0x1a86b24ca84aa7049a974bb6db83f0bd80c60713
Symbol: DTCN
Decimals: 18
Supply: 220,000,000
Website: https://www.dcoin.us/
Official Email: info@dcoin.us
Pull Request:
https://github.com/decentrex/decentrex.github.io/pull/42

It would be my pleasure to receive a reply from yours.

Thank you so much,

Yours sincerely
DCoin Team

**Comment by amirdcn** [comment]
Please add DCoin (DTCN) on DECENTREX.

**Comment by robinstd** [comment]
This is the best ERC20 token. 
They are active and dedicated. You can really see their determination to be successful and they are one step ahead.
So Please add DTCN to DECENTREX.
Use appropriate hashtag to support DCoin

#support
#DCoin  #DTCN

**Comment by jakson007** [comment]
#support #DCoin @DTCN 
This is the time to do something revolutionary to make an ultimate victory.

I think it should be add to the decentrex. 🚀
So my cordially request to Add it to DECENTREX!
Thanks
Jakson

**Comment by Alfredpinto** [comment]
I like DCoin. I think it will be boom in future for sure.

**Comment by saimonfe** [comment]
DCoin has been doing great since it started with a big and strong community and it would be great to see DCoin being traded in some exchanges and the whole community would be grateful if decentrex exchange would list DCoin.

**Comment by rinorious** [comment]
Please support DCoin
This coin doing great.

**Comment by robert312** [comment]
Decentrex team kindly please add DCoin token in your exchange. The community will be very happy if you guys consider adding this token. 
Thank you

**Comment by hanson111** [comment]
As a trader I would like to see my holdings in a platform where I can freely exchange from one crypto to another. I am a holder of this token and DCoin has active community. We are looking forward for DCoin token to be listed on Decentrex.

**Comment by manrikpal** [comment]
Decentrex help us to add DCoin to your exchange option. It really have a good background.

**Comment by randy9091** [comment]
DCoin have a great and clear goal. A promising success!!
Please add DCoin at decentrex list.

**Comment by palmalde** [comment]
Many more people already become fan of DCoin and want to walk with this long run.
I think Decentrex should add this on the lists.

Thanks
Melleisou Palmal

**Comment by steward1994** [comment]
Really nice DTCN.

**Comment by Albruuzee** [comment]
We want dcoin enlisted in decentrex very soon.
plz decentrex team add dtcn very soon

**Comment by chrisfox99** [comment]
i love dcoin.
please add dcoin to the decentrex list.

**Comment by chrisfox99** [comment]
#support #DCoin #DTCN

**Comment by qubore** [comment]
Please Support This Token !!
That will be one of the highest value of token ! :) I'm sure of it!

**Comment by Deximar** [comment]
I love DCoin and i am damn sure that will be boom very soon. so please add it to decentrex

**Comment by raymond1217** [comment]
DCoin has been doing great since it started with a big and strong community and it would be great to see DTCN being traded in some exchanges and the whole community would be grateful if decentrex exchange would list DCoin.

**Comment by omniatrader** [comment]
I am cryptocurrency investor and trader. Doing well last few years. But now i am really astonisehd about DCoin future. No doubt it will be the highest value in short time. 
Please add #DCoin to the Decentrex.
#support #DTCN

**Comment by kapoex** [comment]
Wow tons of fake user spam in here

**Comment by kapoex** [comment]
i would not add this scam coin

**Comment by dtcn** [comment]
@kapoex Are you the Author of the DECENTREX? If no, then you are not in the right place. If I'm not wrong, you are going to start a new Exchange named Kapoex, but we're not interested to add our upcoming DCoin into your Kapoex Exchange!!!
You can't imagine, how big community DCoin have. But you are most welcome to know details about DCoin and we are cordially invite you to give any feedback, advise, guide to improve DCoin.
Thanks

**Comment by micky69** [comment]
dtcn is a great token
i've got all the reason to add

**Comment by kapoex** [comment]
@dtcn, my clone of decentrex is by no means ready yet but we wont add you anyways. I was referring to the fake user spam you are making HERE. You arent fooling anyone, these are not "community members" they are you, spamming with bot or fake shill accounts. Good day.

**Comment by 5mil** [comment]
"DCoin is a virtual digital currency project established on the basis of Decentralization method of Bitcoin Agreement" completely false... "DCoin" isnt even a coin, its a token on expanse so it hasnt "established" any decentralization itself.

**Comment by 5mil** [comment]
All of the accounts promoting this "Dcoin" were created minutes before posting here. Clear scam.

**Comment by dtcn** [comment]
@5mil Yes, you are right! "DCoin" is a token on eth. We've edited/ updated this information. 
Again, You are right, all of the accounts promoting this "DCoin" were created minutes before posting here. Some of the peoples from our community, just signup to Github and start commenting. We already boosted the news/information to stop commenting. They're followed the previous issue's of EXMR Token:
https://github.com/decentrex/decentrex.github.io/issues/30
All of the 41 Comments from fresh new accounts.

Anyway, No one will comments from our community. Most the Token is listed into the DECENTREX without create any issue and pull request. Its totally up to DECENTREX, DCoin (DTCN) will be added or not. But our community still can trade/ exchange DCoin (DTCN) into the DECENTREX:
https://decentrex.com/#0x1a86b24ca84aa7049a974bb6db83f0bd80c60713-ETH

If DCoin (DTCN) can meet the daily trade/ exchange volume and then DECENTREX may be decided to add DCoin (DTCN) into the list.

*No Need any Pull Request or Issues*

Many Many Thanks!

**Comment by CRYPNM** [comment]
Hi DESCENTREX, please consider listing my token:

Token address: 0x7efa60b81287f7e609495833a6ce5475c8441bb7

ETHERSCAN: https://etherscan.io/address/0x7efa60b81287f7e609495833a6ce5475c8441bb7

Official Web site: smartnode.org

WHITE PAPER: http://smartnode.org/main/WhitePaper-English.pdf

TOKEN SPECS:

NAME: SMART NODE, SYMBOL: NODE, DECIMALS: 0,

Total Supply 40.000.000

Social:

TWITTER https://twitter.com/SmartNodeGroup

FACEBOOK https://www.facebook.com/smart.node.5

INSTAGRAM https://www.instagram.com/smartnodegroup/

TELEGRAM https://t.me/joinchat/AAAAAFHcOBf0FBlKN0DbOQ

DISCORD GROUP https://discord.gg/sfc7rEN

THANKS!
